front/Dockerfile

```diff
@@ -27,6 +27,9 @@ FROM nginx:1.21.6-alpine
 COPY front/nginx.conf /etc/nginx/conf.d/default.conf
 COPY front/templater.sh /
 
+RUN apt install -y gettext
+RUN envsubst < server.template.json > dist/server.json
```

Comment:
> No need to install `gettext`, it is already installed.

Are you sure about that ? Because on Node builder container, the lib is not installed.